docs/source/conf.py

```diff
@@ -51,6 +51,7 @@
     "sphinx.ext.inheritance_diagram",
     "sphinxcontrib.programoutput",
     "myst_parser",
+    "sphinx_togglebutton"
```

Comment:
Has this dependency been added to the poetry dependencies?

Reply:
This should be fixed in the first commit, it was my fault as I wasn't using poetry so I didn't add it to the dependenies